tests/core/data/all_config.toml

```diff
@@ -22,5 +22,4 @@ pft_name = "broadleaf"
 maxh = 50.0
 
 [soil]
-no_layers = 1
```

Comment:
What's happened to `no_layers`?

Reply:
No longer planing for the `soil` module to use multiple soil layers, was basically told that this would be complete and utter nightmare to parameterise, so switching to a simpler model where there's a single homogenous layer where all the processes of interest happens. The `abiotic` model is planning to keep discrete soil layers though for calculation of energy and water transfers through soil (which I *think* are a bit easier to parameterise).

Reply:
Ok cool. I was just curious!